### PR TITLE
issues fixes

### PR DESCRIPTION
**Pull request by tachella**
Fixing issues #192, #174 and #207 

### Checks to be done before submitting your PR
- [x] `python3 -m pytest tests/` runs successfully.
- [x] `black .` runs successfully.
- [x] `make html` runs successfully (in the `docs/` directory).
- [x] Updated docstrings related to the changes (as applicable).
- [ ] Added an entry to the [CHANGELOG.rst](../CHANGELOG.rst).

### CODE REVIEW

File: deepinv/physics/generator/mri.py [hunk 1]
```diff
@@ -39,35 +39,45 @@ def step(self, batch_size=1):
         Create a mask of vertical lines.
 
         :param int batch_size: batch_size.
-        :return: dictionary with key **'mask'**: tensor of size (batch_size, 1, H, W) with values in {0, 1}.
+        :return: dictionary with key **'mask'**: tensor of size (batch_size, C, H, W) with values in {0, 1}.
```
Comment (matthieutrs) [inline deepinv/physics/generator/mri.py:42]:
```suggestion
        :return: dictionary with key `mask`: tensor of size (batch_size, C, H, W) with values in {0, 1}.
```

File: deepinv/physics/range.py [hunk 2]
```diff
@@ -8,61 +8,46 @@ class Decolorize(DecomposablePhysics):
...
+        self.mask = torch.nn.Parameter(torch.ones((1), device=device) * 0.66851)
 
     def V_adjoint(self, x):
-        y = x[:, 0, :, :] * 0.2989 + x[:, 1, :, :] * 0.5870 + x[:, 2, :, :] * 0.1140
+        y = x[:, 0, :, :] * 0.4472 + x[:, 1, :, :] * 0.8781 + x[:, 2, :, :] * 0.1706
         return y.unsqueeze(1)
 
     def V(self, y):
-        return torch.cat([y * 0.2989, y * 0.5870, y * 0.1140], dim=1)
+        return torch.cat([y * 0.4472, y * 0.8781, y * 0.1706], dim=1)
```
Comment (matthieutrs) [inline deepinv/physics/range.py:39]: Why multiplying the mask with a normalization factor? I get the two are equivalent but (a) this does not change the norm of the operator (b) in both cases we have perfect adjointness.

Comment (tachella) [reply deepinv/physics/range.py:39]: it was to make sure that V is an orthonormal operation

**Review by matthieutrs — APPROVED** [review]
